plugin-rapl/src/powercap.rs

```diff
@@ -17,24 +18,16 @@ use alumet::{
 use alumet::{metrics::TypedMetricId, pipeline::elements::error::PollError};
 use anyhow::{anyhow, Context};
 
-use super::domains::RaplDomainType;
-
-const POWERCAP_RAPL_PATH: &str = "/sys/devices/virtual/powercap/intel-rapl";
+pub const POWERCAP_RAPL_PATH: &str = "/sys/devices/virtual/powercap/intel-rapl";
 const POWER_ZONE_PREFIX: &str = "intel-rapl";
 const POWERCAP_ENERGY_UNIT: f64 = 0.000_001; // 1 microJoules
 
 const PERMISSION_ADVICE: &str = "Try to adjust file permissions.";
 
-/// Hierarchy of power zones
-pub struct PowerZoneHierarchy {
```

Comment:
Can you keep this one? I want to add a feature that shows the domain hierarchy, later.